### DISCUSSION TITLE
do you have an example or tutorial how to use your library with nextjs14 for face detection?

### DISCUSSION THREAD

**Discussion by thenotoriousgustav** [vladmandic/human#453] [Q&A]
i have an skill issue, because there is a problem with tfjs-node, i dont understand, im using a client side for this project. it would be nice if you can give me a tutorial for using this library with nextjs14

**Comment by vladmandic** [comment]
if you're using nextjs, by default it will try to compile everything server-side and that means that library will autodetect that it runs in nodejs environment, not client-side. you need to explicitly load library client-side.

this is quite an old example so things have changed with nextjs since then, but its a good starting point: https://github.com/vladmandic/human-next
